lib/ember_cli/path_set.rb

```diff
@@ -63,18 +67,19 @@ def build_error_file
     def bower
       @bower ||= begin
         bower_path = app_options.fetch(:bower_path) { which("bower") }
+        bower_executable = Pathname(bower_path.to_s)
```

Comment:
I was surprised this works! Is there any notable difference between `Pathname.new` and `Pathname()`.